.github/workflows/ci.yml

```diff
@@ -15,17 +15,27 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: actions/checkout@v4
+      - name: Get Core Sans
+        uses: actions/checkout@v4
+        if: "!github.event.pull_request.head.repo.fork"
         with:
           fetch-depth: 0
           repository: mainmatter/core-sans-a-fonts
           ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
           path: core-sans-a-fonts
-      - name: Install Fonts
+      - name: Install Core Sans Font
+        if: "!github.event.pull_request.head.repo.fork"
         run: |
           sudo cp -r core-sans-a-fonts/* /usr/local/share/fonts/
           sudo fc-cache -f -v
           fc-list | grep "Core Sans"
+      - name: Use Fallback font for fork PRs
+        if: "github.event.pull_request.head.repo.fork"
+        run: |
+          sed -i 's/"BoldFont=CoreSansA65.ttf",//g' book/book.toml
+          sed -i 's/"ItalicFont=CoreSansA45It.ttf",//g' book/book.toml
+          sed -i 's/"BoldItalicFont=CoreSansA65It.ttf",//g' book/book.toml
+          sed -i 's/CoreSansA45.ttf/Open Sans:style=Regular/g' book/book.toml
```

Comment:
I'm not really too happy with this but it's the best way I found to fix this without massive refactorings